meminfo.c

```diff
@@ -76,9 +85,21 @@ meminfo_t parse_meminfo()
     }
     rewind(fd);
 
-    size_t len = fread(buf, 1, sizeof(buf) - 1, fd);
-    if (len == 0) {
-        fatal(102, "could not read /proc/meminfo: 0 bytes returned\n");
+    // Loop to handle short reads.
```

Comment:
Remove retry loop, glibc handles this internally

Reply:
Adjusted as per https://github.com/rfjakob/earlyoom/issues/189#issuecomment-619587954